lib/src/components/Select/select.css.ts

```diff
@@ -153,12 +151,27 @@ export const inputRecipe = recipe({
         "::placeholder": { color: "#DA2C2C" },
       },
     },
+    isMulti: {
+      true: {
+        width: 0,
+        minWidth: "50px",
+        flex: 1,
+        fontSize: "14px",
+      },
+    },
   },
 });
 
 export const inputTextContainer = style({
-  width: "85%",
+  width: "80%",
   fontSize: "16px",
   whiteSpace: "nowrap",
   overflow: "hidden",
+  height: "100%",
+});
+
+globalStyle(`${selectContainerStyles} [data-hover="true"]`, {
```

Comment:
Move this to a separate file called .glibalstyles.css.ts

Reply:
sure